pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -601,7 +614,7 @@ def select_precision_plugin(self) -> PrecisionPlugin:
                 log.info("Using APEX 16bit precision.")
                 return ApexMixedPrecisionPlugin(self.amp_level)
 
-        raise NotImplementedError("We only support precisions 64, 32 and 16!")
+        raise NotImplementedError(f"We only support precision: {PrecisionType.supported_types()}")
```

Comment:
this won't ever be raised since you already check this in `_validate_precision_type`, correct?

Inline the code of `_validate_precision_type` here?

Reply:
Yep exactly, I'm unsure what to do here... any suggestions?